heroku

```diff
@@ -0,0 +1,5 @@
+<html>
+<body>
+	You are being <a href="https://heroku.com/deploy?template=https://github.com/tchan408/lex-connector/blob/patch-2">redirected</a>
```

Comment:
Does this need updating for us to merge this to master?

Reply:
Yes, necessary update has been done. Thank you